MekHQ/data/rat/(3026) House Arano/Capellan Confederation/Aerospace/DropShip/3026 Capellan Confederation Aerospace - DropShip.txt

```diff
@@ -0,0 +1,12 @@
+3026 Capellan Confederation Aerospace - DropShip
```

Comment:
The CapCon is pretty well covered for RATs in the 3025 era, is there a reason we need another one?

Reply:
It is included in the Arano RAT, so that allows people to just play the Aurigan Reach? Also, just adding the whole RAT, not parts of it.

Reply:
I'm not sure that the RAT mechanism really takes into account the current location. You just tell it a faction and a year, and it spits out units. But I suppose if people don't want to use these new ones, they don't have to select them.